tests/muxcable_test.py

```diff
@@ -0,0 +1,408 @@
+import os
+import sys
+import traceback
+
+import mock_tables.dbconnector
+from click.testing import CliRunner
+from unittest import mock
+from utilities_common.db import Db
+
+sys.modules['sonic_platform_base'] = mock.Mock()
+sys.modules['sonic_platform_base.sonic_sfp'] = mock.Mock()
+sys.modules['sonic_platform_base.sonic_sfp.sfputilhelper'] = mock.Mock()
+#sys.modules['platform_sfputil'] = mock.Mock()
+import config.main as config
+import show.main as show
+
+
+tabular_data_status_output_expected = """PORT        STATUS    HEALTH
```

Comment:
Suggest changing this to the following for better readability:

```python3
tabular_data_status_output_expected = """\
PORT        STATUS    HEALTH
```

And please do the same for all similar in this file

Reply:
fixed